docs/hdfs_connector.rst

```diff
@@ -152,6 +152,9 @@ specifies the HDFS we are writing data to and you should set this according to y
 The ``flush.size`` specifies the number of records the connector need to write before invoking file
 commits.
 
+.. note:: For HA HDFS deployments you will need to include ``hadoop.conf.dir``, setting it to a directory which includes both hdfs-site.xml. Once hdfs-site.xml is in place and ``hadoop.conf.dir`` has been set ``hdfs.url`` may be set to the namenodes nameservice id. i.e. 'nameservice1' . 
```

Comment:
Yep sorry, the 'both' was still there from when I thought I needed core-site.xml as well. Fixing it now